libobs/audio-monitoring/osx/coreaudio-output.c

```diff
@@ -286,6 +297,14 @@ static void audio_monitor_init_final(struct audio_monitor *monitor)
 	if (monitor->ignore)
 		return;
 
+	monitor->audio_track = (monitor->source->info.output_flags &
+				OBS_SOURCE_AUDIO_TRACK) != 0;
+
+	if (monitor->audio_track)
+		os_atomic_inc_long(&audio_track_active);
+	else
+		os_atomic_dec_long(&audio_track_active);
```

Comment:
When an instance is created, is this intentional to decrement `audio_track_active`?

And, when destroying the instance, does `audio_track_active` decremented?

Reply:
This is fixed with latest update